examples/ex19p.cpp

```diff
@@ -486,8 +486,8 @@ void JacobianPreconditioner::SetOperator(const Operator &op)
 
       if (!spaces[0]->GetParMesh()->Nonconforming())
       {
-#ifndef HYPRE_USING_CUDA
-         // Not available yet when hypre is built with CUDA
+#if !defined(HYPRE_USING_GPU)
+         // Not available yet when hypre is built with GPU
```

Comment:
I don't understand this comment, I think we should either make it more clear or remove it.